tox.ini

```diff
@@ -1,8 +1,14 @@
 [tox]
-envlist = py26,py27,py33,py34,py35,py36
+envlist = py{26,27,33,34,35,36}-{chardet,cchardet}
 
 [testenv]
 
+deps =
+    py{26,27,33,34,35,36}-chardet: chardet
```

Comment:
I'm not quite sure, how I can test against ccharset and charset using existing cases